recipes/libcurl/all/conanfile.py

```diff
@@ -472,150 +506,89 @@ def _arm_version(self, arch):
             version = int(match.group(1))
         return version
 
-    def _build_with_autotools(self):
-        with chdir(self, self._source_subfolder):
-            # autoreconf
-            self.run("{} -fiv".format(tools.get_env("AUTORECONF") or "autoreconf"), win_bash=tools.os_info.is_windows, run_environment=True)
-
-            # fix generated autotools files to have relocatable binaries
-            if is_apple_os(self):
-                replace_in_file(self, "configure", "-install_name \\$rpath/", "-install_name @rpath/")
-
-            self.run("chmod +x configure")
-
-            # run configure with *LD_LIBRARY_PATH env vars it allows to pick up shared openssl
-            with tools.run_environment(self):
```

Comment:
It seems there is still an issue here. See https://github.com/conan-io/conan-center-index/pull/12377#discussion_r962293843